internal/indexer/definitions/torrentday.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#id: torrentday
+name: TorrentDay
+identifier: td
+description: TorrentDay (TD) is a private torrent tracker for GENERAL.
+language: en-us
+urls:
+  - https://www.torrentday.com
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: passkey
+    type: secret
+    label: Passkey
+    help: "Check the tracker tab in your torrent client to find your passkey"

```

Comment:
Is it not visible anywhere in the profile or profile settings? If not that's fine and this can stay.

Reply:
Have updated the help, we can indeed find it by creating a RSS link